forwarder/statistics_reporter.py

```diff
@@ -0,0 +1,26 @@
+import graphyte  # type: ignore
+
+
+class StatisticsReporter:
+    def __init__(
+        self,
+        graphyte_server: str,
+        logger,
+        prefix: str = "throughput",
```

Comment:
Perhaps check with @amues to see if he has a particular prefix suggestion.

Reply:
Cool. He is in office but off for lunch. Will catch him for this today.

Reply:
It probably needs to identify this Forwarder instance in the prefix at least, maybe by using the service_id?
https://github.com/ess-dmsc/forwarder/blob/7399b1b8da9fdd4ae917a82af50f9506729a2d88/forwarder_launch.py#L29

Reply:
he suggests `forwarder.throughput`

Reply:
How about  `f"forwarder.{args.service_id}.throughput"`